src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/config.json

```diff
@@ -0,0 +1,126 @@
+﻿{
+  "properties": {
+    "OSGroup": {
+      "explanation": "Target OS for result binaries.",
+      "type": "property",
+      "values": "Windows_NT, Linux, OSX, FreeBSD, NetBSD",
```

Comment:
Shouldn't "values" be an array? What if one of my "values" is supposed to have a comma in it? Seems like we'll get into a weird well of escaping otherwise.